ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -413,5 +413,95 @@ ViewData["Title"] = "Notification - Clinical Details";
             </div>
         </div>
     </date-comparison>
+    <br/>
+
+    <h2>Treatment Options</h2>
+    <validate-multiple model="ClinicalDetails" :properties="['IsShortCourseTreatment', 'IsMDRTreatment']" inline-template>
+        <div>
+             @{ var ShortCourseGroupState = !Model.IsValid("ClinicalDetails.IsShortCourseTreatment") ? Error : Standard; }
+            <nhs-form-group nhs-form-group-type="@ShortCourseGroupState" v-bind:class="{ 'nhsuk-form-group--error': hasError}">
+                <nhs-fieldset>
+                    <nhs-fieldset-legend nhs-legend-size="Standard">Has Started Short Course Treatment?</nhs-fieldset-legend>
+                    <span ref="errorField[0]" nhs-span-type="ErrorMessage" asp-validation-for="ClinicalDetails.IsShortCourseTreatment"></span>
+                    <nhs-radios nhs-radios-type="Standard">
+                        <nhs-radios-item>
+                            <input nhs-input-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment" id="short-course-yes" type="radio" value="true" ref="input[0]" v-on:change="validate"/>
+                            <label nhs-label-type="Radios" for="short-course-yes">Yes</label>
+                        </nhs-radios-item>
+                        <nhs-radios-item>
+                            <input nhs-input-type="Radios" asp-for="ClinicalDetails.IsShortCourseTreatment" id="short-course-no" type="radio" value="false" v-on:change="validate"/>
+                            <label nhs-label-type="Radios" for="short-course-no">No</label>
+                        </nhs-radios-item>
+                    </nhs-radios>
+                </nhs-fieldset>
+            </nhs-form-group>
+            @{var mdrTreatmentErrorClass = !Model.IsValid("ClinicalDetails.IsMDRTreatment") ? errorClassName : ""; }
+            <div class="nhsuk-form-group @mdrTreatmentErrorClass" v-bind:class="{ 'nhsuk-form-group--error': hasError}" aria-describedby="mdr-treatment-error">
+                <fieldset class="nhsuk-fieldset">
```

Comment:
can we use `nhsuk-fieldset`? (also for `legend`, `error` and `radios` below)

Reply:
sorry this was just copied from a different conditionally revealing fieldset where we're still using the old formatting - I'll try and fix this elsewhere too as part of this commit!

Reply:
thanks!